elastic_training/README.md

```diff
@@ -0,0 +1,25 @@
+Use the following script to start elastic training with 2 node * 8 gpus:
+
+Node 1: *(IP: 192.168.1.1, and has a free port: 1234)*
+
+```bash
+python3 -m bagua.distributed.run \
+        --nnodes=1:4 \
```

Comment:
2 nodes, but nnodes=1:4, is this expected?

Reply:
Yes it is. document changed.